other-builds/ndkbuild/gles3jni/app/build.gradle

```diff
@@ -0,0 +1,50 @@
+/*  This build is set for platform support ES3.
+ *  to build for ES2:
+ *      change android:glEsVersion="0x00030000 in AndroidManifest.txt to 0x00020000
+ *      minSDVersion to  sth less than 11
+ *      command line argument APP_PLATFORM=android-11 (or lower)
+ */
+
+apply plugin: 'com.android.application'
+
+// pointing to cmake's source code for the same project
+def REMOTE_SRC_ROOT = '../../../../' + rootProject.getName() + '/' +
+                       project.getName() + '/src/main'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion '23.0.2'
+
+    defaultConfig {
+        applicationId 'com.android.gles3jni'
+        minSdkVersion 18
+        targetSdkVersion 11
+        ndkConfig.abiFilters 'x86', 'x86_64', 'armeabi', 'armeabi-v7a', 'arm64-v8a'
+        externalNativeBuild {
+            ndkBuild {
+                arguments  'APP_PLATFORM=android-18'
```

Comment:
I don't think this APP_PLATFORM argument needs to be here anymore (and it's not a great example to set for user)